server/controllers/apply/applicationsController.ts

```diff
@@ -52,7 +53,7 @@ export default class ApplicationsController {
   }
 
   private ineligibleViewParams(application: Cas2Application): Record<string, string | Cas2Application> {
-    const panelText = `${application.person.name} is not eligible for CAS-2 accommodation`
+    const panelText = `${nameOrPlaceholderCopy(application.person)} is not eligible for CAS-2 accommodation`
```

Comment:
how placeholder is the text? (is it worth capitalising 'the' here?)

Reply:
hopefully the user should never see the placeholder text in CAS2 because we shouldn't have an application with an LAO...  but I think we need to account for the person becoming an LAO during an application? And because of the types now coming from API we always need to check this... 

tbh I think it the solution could be looked at again with a bit more time as I'm not 100% sure we are handling everything in the best way, but want to unblock us from being able to update API types now.

Reply:
ok yep, makes sense

Reply:
think I have now capitalised wherever the name is the first word of a sentence